app/src/main/java/com/github/factotum_sdp/factotum/ui/login/LoginViewModel.kt

```diff
@@ -41,11 +38,7 @@ class LoginViewModel(private val userViewModel: UserViewModel) : BaseAuthViewMod
                     LoginResult(success = result.data)
             } else {
                 _loginResult.value = LoginResult(
-                    error = if ((result as Result.Error).exception.message == "User not found") {
```

Comment:
Should maybe not be pushed since this modification  purpose is for testing when login is down

Reply:
Sure, I put the right version back